test/src/edu/stanford/nlp/util/AcronymMatcherTest.java

```diff
@@ -20,15 +20,6 @@ public void testBasic() {
     assertTrue(AcronymMatcher.isAcronym("ECC", "Election Complaints Commission".split("\\s+")));
   }
 
-  @Test
```

Comment:
I think this change can be dropped, right?  It was a test added by your test

Reply:
What I meant is, I merged that test.  Do you think it's better to remove it?  We can, if you like